config/manager/manager.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       - command:
         - /manager
         args:
-        - --leader-elect
+        - --leader-elect=false
```

Comment:
curious to what caused this thing to appear here

Reply:
lets chat about this in zoom